sdk/documentintelligence/azure-ai-documentintelligence/README.md

```diff
@@ -525,6 +574,33 @@ from azure.core.credentials import AzureKeyCredential
 from azure.ai.documentintelligence import DocumentIntelligenceClient
 from azure.ai.documentintelligence.models import AnalyzeResult
 
+def print_table(header_names, table_data):
+    """Print a two-dimensional array like a table.
+
+    Based on provided column header names and two two-dimensional array data, print the strings like table.
+
+    Args:
+        header_names: An array of string, it's the column header names.  e.g. ["name", "gender", "age"]
+        table_data: A two-dimensional array, they're the table data.  e.g. [["Mike", "M", 25], ["John", "M", 19], ["Lily", "F", 23]]
+    Return: None
```

Comment:
Where is this function copied from? IMO the docs for this function take attention away from the main content of the sample.

Reply:
It was added in last extracting tables change: https://github.com/Azure/azure-sdk-for-python/commit/b5c8353051e1927650c45db097f173d8bcb3ad8d#diff-c9241c8efaf0e7d0357f789089f921f08d97ede06c27c916d08538fd3735d630
We can just have one sentence description here, like other comments.

Reply:
Do you know where that code came from? It was a community contribution?

Reply:
It was added by a vendor on helping a customer issue for extracting tables, not from external contributor.